---
title: "Ribosomes and Protein Synthesis"
layout: page
---


<div data-type="abstract" markdown="1">
By the end of this section, you will be able to:
* Describe the different steps in protein synthesis
* Discuss the role of ribosomes in protein synthesis

</div>

The synthesis of proteins consumes more of a cell’s energy than any other metabolic process. In turn, proteins account for more mass than any other component of living organisms (with the exception of water), and proteins perform virtually every function of a cell. The process of translation, or protein synthesis, involves the decoding of an mRNA message into a polypeptide product. Amino acids are covalently strung together by interlinking peptide bonds in lengths ranging from approximately 50 amino acid residues to more than 1,000. Each individual amino acid has an amino group (NH<sub>2</sub>) and a carboxyl (COOH) group. Polypeptides are formed when the amino group of one amino acid forms an amide (i.e., peptide) bond with the carboxyl group of another amino acid ([\[link\]](#fig-ch15_05_01)). This reaction is catalyzed by ribosomes and generates one water molecule.

![Illustration shows two amino acids side-by-side. Each amino acid has an amino group, a carboxyl group, and a side chain labeled R or R\'. Upon formation of a peptide bond, the amino group is joined to the carboxyl group. A water molecule is released in the process.](../resources/Figure_15_05_01.jpg "A peptide bond links the carboxyl end of one amino acid with the amino end of another, expelling one water molecule. For simplicity in this image, only the functional groups involved in the peptide bond are shown. The R and R' designations refer to the rest of each amino acid structure."){: #fig-ch15_05_01 data-media-type="image/jpg"}

# The Protein Synthesis Machinery

In addition to the mRNA template, many molecules and macromolecules contribute to the process of translation. The composition of each component may vary across species; for instance, ribosomes may consist of different numbers of rRNAs and polypeptides depending on the organism. However, the general structures and functions of the protein synthesis machinery are comparable from bacteria to human cells. Translation requires the input of an mRNA template, ribosomes, tRNAs, and various enzymatic factors.

<div data-type="note" class="interactive" data-label="" markdown="1">
<div data-type="title">
Link to Learning
</div>
<span data-type="media" data-alt="QR Code representing a URL"> ![QR Code representing a URL](../resources/prokary_protein.png){: data-media-type="image/png"} </span>
Click through the steps of this [PBS interactive][1]{: target="_window"} to see protein synthesis in action.

</div>

## Ribosomes

Even before an mRNA is translated, a cell must invest energy to build each of its ribosomes. In *E. coli*, there are between 10,000 and 70,000 ribosomes present in each cell at any given time. A ribosome is a complex macromolecule composed of structural and catalytic rRNAs, and many distinct polypeptides. In eukaryotes, the nucleolus is completely specialized for the synthesis and assembly of rRNAs.

Ribosomes exist in the cytoplasm in prokaryotes and in the cytoplasm and rough endoplasmic reticulum in eukaryotes. Mitochondria and chloroplasts also have their own ribosomes in the matrix and stroma, which look more similar to prokaryotic ribosomes (and have similar drug sensitivities) than the ribosomes just outside their outer membranes in the cytoplasm. Ribosomes dissociate into large and small subunits when they are not synthesizing proteins and reassociate during the initiation of translation. In *E. coli*, the small subunit is described as 30S, and the large subunit is 50S, for a total of 70S (recall that Svedberg units are not additive). Mammalian ribosomes have a small 40S subunit and a large 60S subunit, for a total of 80S. The small subunit is responsible for binding the mRNA template, whereas the large subunit sequentially binds tRNAs. Each mRNA molecule is simultaneously translated by many ribosomes, all synthesizing protein in the same direction: reading the mRNA from 5\' to 3\' and synthesizing the polypeptide from the N terminus to the C terminus. The complete mRNA/poly-ribosome structure is called a **polysome**{: data-type="term"}.

## tRNAs

The tRNAs are structural RNA molecules that were transcribed from genes by RNA polymerase III. Depending on the species, 40 to 60 types of tRNAs exist in the cytoplasm. Serving as adaptors, specific tRNAs bind to sequences on the mRNA template and add the corresponding amino acid to the polypeptide chain. Therefore, tRNAs are the molecules that actually “translate” the language of RNA into the language of proteins.

Of the 64 possible mRNA codons—or triplet combinations of A, U, G, and C—three specify the termination of protein synthesis and 61 specify the addition of amino acids to the polypeptide chain. Of these 61, one codon (AUG) also encodes the initiation of translation. Each tRNA anticodon can base pair with one of the mRNA codons and add an amino acid or terminate translation, according to the genetic code. For instance, if the sequence CUA occurred on an mRNA template in the proper reading frame, it would bind a tRNA expressing the complementary sequence, GAU, which would be linked to the amino acid leucine.

As the adaptor molecules of translation, it is surprising that tRNAs can fit so much specificity into such a small package. Consider that tRNAs need to interact with three factors: 1) they must be recognized by the correct aminoacyl synthetase (see below); 2) they must be recognized by ribosomes; and 3) they must bind to the correct sequence in mRNA.

## Aminoacyl tRNA Synthetases

The process of pre-tRNA synthesis by RNA polymerase III only creates the RNA portion of the adaptor molecule. The corresponding amino acid must be added later, once the tRNA is processed and exported to the cytoplasm. Through the process of tRNA “charging,” each tRNA molecule is linked to its correct amino acid by a group of enzymes called **aminoacyl tRNA synthetases**{: data-type="term"}. At least one type of aminoacyl tRNA synthetase exists for each of the 20 amino acids; the exact number of aminoacyl tRNA synthetases varies by species. These enzymes first bind and hydrolyze ATP to catalyze a high-energy bond between an amino acid and adenosine monophosphate (AMP); a pyrophosphate molecule is expelled in this reaction. The activated amino acid is then transferred to the tRNA, and AMP is released.

# The Mechanism of Protein Synthesis

As with mRNA synthesis, protein synthesis can be divided into three phases: initiation, elongation, and termination. The process of translation is similar in prokaryotes and eukaryotes. Here we’ll explore how translation occurs in *E. coli*, a representative prokaryote, and specify any differences between prokaryotic and eukaryotic translation.

## Initiation of Translation

Protein synthesis begins with the formation of an initiation complex. In *E. coli*, this complex involves the small 30S ribosome, the mRNA template, three initiation factors (IFs; IF-1, IF-2, and IF-3), and a special **initiator tRNA**{: data-type="term"}, called <math xmlns="http://www.w3.org/1998/Math/MathML" display=""> <mrow> <mtext>t</mtext><mtext>R</mtext><mtext>N</mtext><msubsup> <mtext>A</mtext> <mtext>f</mtext> <mrow> <mtext>M</mtext><mtext>e</mtext><mtext>t</mtext> </mrow> </msubsup> </mrow> </math>

. The initiator tRNA interacts with the **start codon**{: data-type="term"} AUG (or rarely, GUG), links to a formylated methionine called fMet, and can also bind IF-2. Formylated methionine is inserted by <math xmlns="http://www.w3.org/1998/Math/MathML" display=""> <mrow> <mtext>f</mtext><mtext>M</mtext><mtext>e</mtext><mtext>t</mtext><mo>−</mo><mtext>t</mtext><mtext>R</mtext><mtext>N</mtext><msubsup> <mtext>A</mtext> <mtext>f</mtext> <mrow> <mtext>M</mtext><mtext>e</mtext><mtext>t</mtext> </mrow> </msubsup> </mrow> </math>

 at the beginning of every polypeptide chain synthesized by *E. coli*, but it is usually clipped off after translation is complete. When an in-frame AUG is encountered during translation elongation, a non-formylated methionine is inserted by a regular Met-tRNA<sup>Met</sup>.

In *E. coli* mRNA, a sequence upstream of the first AUG codon, called the **Shine-Dalgarno sequence**{: data-type="term"} (AGGAGG), interacts with the rRNA molecules that compose the ribosome. This interaction anchors the 30S ribosomal subunit at the correct location on the mRNA template. Guanosine triphosphate (GTP), which is a purine nucleotide triphosphate, acts as an energy source during translation—both at the start of elongation and during the ribosome’s translocation.

In eukaryotes, a similar initiation complex forms, comprising mRNA, the 40S small ribosomal subunit, IFs, and nucleoside triphosphates (GTP and ATP). The charged initiator tRNA, called Met-tRNA<sub>i</sub>, does not bind fMet in eukaryotes, but is distinct from other Met-tRNAs in that it can bind IFs.

Instead of depositing at the Shine-Dalgarno sequence, the eukaryotic initiation complex recognizes the 7-methylguanosine cap at the 5\' end of the mRNA. A cap-binding protein (CBP) and several other IFs assist the movement of the ribosome to the 5\' cap. Once at the cap, the initiation complex tracks along the mRNA in the 5\' to 3\' direction, searching for the AUG start codon. Many eukaryotic mRNAs are translated from the first AUG, but this is not always the case. According to **Kozak’s rules**{: data-type="term"}, the nucleotides around the AUG indicate whether it is the correct start codon. Kozak’s rules state that the following consensus sequence must appear around the AUG of vertebrate genes: 5\'-gccRccAUGG-3\'. The R (for purine) indicates a site that can be either A or G, but cannot be C or U. Essentially, the closer the sequence is to this consensus, the higher the efficiency of translation.

Once the appropriate AUG is identified, the other proteins and CBP dissociate, and the 60S subunit binds to the complex of Met-tRNA<sub>i</sub>, mRNA, and the 40S subunit. This step completes the initiation of translation in eukaryotes.

## Translation, Elongation, and Termination

In prokaryotes and eukaryotes, the basics of elongation are the same, so we will review elongation from the perspective of *E. coli*. The 50S ribosomal subunit of <em>E. coli </em>consists of three compartments: the A (aminoacyl) site binds incoming charged aminoacyl tRNAs. The P (peptidyl) site binds charged tRNAs carrying amino acids that have formed peptide bonds with the growing polypeptide chain but have not yet dissociated from their corresponding tRNA. The E (exit) site releases dissociated tRNAs so that they can be recharged with free amino acids. There is one exception to this assembly line of tRNAs: in *E. coli*, <math xmlns="http://www.w3.org/1998/Math/MathML"> <mrow> <mtext>f</mtext><mtext>M</mtext><mtext>e</mtext><mtext>t</mtext><mo>−</mo><mtext>t</mtext><mtext>R</mtext><mtext>N</mtext><msubsup> <mtext>A</mtext> <mtext>f</mtext> <mrow> <mtext>M</mtext><mtext>e</mtext><mtext>t</mtext> </mrow> </msubsup> </mrow> </math>

 is capable of entering the P site directly without first entering the A site. Similarly, the eukaryotic Met-tRNA<sub>i</sub>, with help from other proteins of the initiation complex, binds directly to the P site. In both cases, this creates an initiation complex with a free A site ready to accept the tRNA corresponding to the first codon after the AUG.

During translation elongation, the mRNA template provides specificity. As the ribosome moves along the mRNA, each mRNA codon comes into register, and specific binding with the corresponding charged tRNA anticodon is ensured. If mRNA were not present in the elongation complex, the ribosome would bind tRNAs nonspecifically.

Elongation proceeds with charged tRNAs entering the A site and then shifting to the P site followed by the E site with each single-codon “step” of the ribosome. Ribosomal steps are induced by conformational changes that advance the ribosome by three bases in the 3\' direction. The energy for each step of the ribosome is donated by an elongation factor that hydrolyzes GTP. Peptide bonds form between the amino group of the amino acid attached to the A-site tRNA and the carboxyl group of the amino acid attached to the P-site tRNA. The formation of each peptide bond is catalyzed by **peptidyl transferase**{: data-type="term"}, an RNA-based enzyme that is integrated into the 50S ribosomal subunit. The energy for each peptide bond formation is derived from GTP hydrolysis, which is catalyzed by a separate elongation factor. The amino acid bound to the P-site tRNA is also linked to the growing polypeptide chain. As the ribosome steps across the mRNA, the former P-site tRNA enters the E site, detaches from the amino acid, and is expelled ([\[link\]](#fig-ch15_05_02)). Amazingly, the <em>E. coli </em>translation apparatus takes only 0.05 seconds to add each amino acid, meaning that a 200-amino acid protein can be translated in just 10 seconds.

<div data-type="note" class="art-connection" data-label="" markdown="1">
<div data-type="title">
Art Connection
</div>
![Illustration shows the steps of protein synthesis. First, the initiator tRNA recognizes the sequence AUG on an mRNA that is associated with the small ribosomal subunit. The large subunit then joins the complex. Next, a second tRNA is recruited at the A site. A peptide bond is formed between the first amino acid, which is at the P site, and the second amino acid, which is at the A site. The mRNA then shifts and the first tRNA is moved to the E site, where it dissociates from the ribosome. Another tRNA binds at the A site, and the process is repeated. ](../resources/Figure_15_05_02.png "Translation begins when an initiator tRNA anticodon recognizes a codon on mRNA. The large ribosomal subunit joins the small subunit, and a second tRNA is recruited. As the mRNA moves relative to the ribosome, the polypeptide chain is formed. Entry of a release factor into the A site terminates translation and the components dissociate."){: #fig-ch15_05_02 data-media-type="image/jpg"}



Many antibiotics inhibit bacterial protein synthesis. For example, tetracycline blocks the A site on the bacterial ribosome, and chloramphenicol blocks peptidyl transfer. What specific effect would you expect each of these antibiotics to have on protein synthesis?

Tetracycline would directly affect:

1.  tRNA binding to the ribosome
2.  ribosome assembly
3.  growth of the protein chain
{: data-number-style="lower-alpha"}

Chloramphenicol would directly affect

1.  tRNA binding to the ribosome
2.  ribosome assembly
3.  growth of the protein chain
{: data-number-style="lower-alpha"}

<!-- <link window="new"  target-id="fig-ch15_05_02" document=""/> Tetracycline: a; Chloramphenicol: c. -->

</div>

Termination of translation occurs when a nonsense codon (UAA, UAG, or UGA) is encountered. Upon aligning with the A site, these nonsense codons are recognized by release factors in prokaryotes and eukaryotes that instruct peptidyl transferase to add a water molecule to the carboxyl end of the P-site amino acid. This reaction forces the P-site amino acid to detach from its tRNA, and the newly made protein is released. The small and large ribosomal subunits dissociate from the mRNA and from each other; they are recruited almost immediately into another translation initiation complex. After many ribosomes have completed translation, the mRNA is degraded so the nucleotides can be reused in another transcription reaction.

# Protein Folding, Modification, and Targeting

During and after translation, individual amino acids may be chemically modified, signal sequences may be appended, and the new protein “folds” into a distinct three-dimensional structure as a result of intramolecular interactions. A **signal sequence**{: data-type="term"} is a short tail of amino acids that directs a protein to a specific cellular compartment. These sequences at the amino end or the carboxyl end of the protein can be thought of as the protein’s “train ticket” to its ultimate destination. Other cellular factors recognize each signal sequence and help transport the protein from the cytoplasm to its correct compartment. For instance, a specific sequence at the amino terminus will direct a protein to the mitochondria or chloroplasts (in plants). Once the protein reaches its cellular destination, the signal sequence is usually clipped off.

Many proteins fold spontaneously, but some proteins require helper molecules, called chaperones, to prevent them from aggregating during the complicated process of folding. Even if a protein is properly specified by its corresponding mRNA, it could take on a completely dysfunctional shape if abnormal temperature or pH conditions prevent it from folding correctly.

# Section Summary

The players in translation include the mRNA template, ribosomes, tRNAs, and various enzymatic factors. The small ribosomal subunit forms on the mRNA template either at the Shine-Dalgarno sequence (prokaryotes) or the 5\' cap (eukaryotes). Translation begins at the initiating AUG on the mRNA, specifying methionine. The formation of peptide bonds occurs between sequential amino acids specified by the mRNA template according to the genetic code. Charged tRNAs enter the ribosomal A site, and their amino acid bonds with the amino acid at the P site. The entire mRNA is translated in three-nucleotide “steps” of the ribosome. When a nonsense codon is encountered, a release factor binds and dissociates the components and frees the new protein. Folding of the protein occurs during and after translation.

# Art Connections

<div data-type="exercise">
<div data-type="problem" markdown="1">
[[link]](#fig-ch15_05_02) Many antibiotics inhibit bacterial protein synthesis. For example, tetracycline blocks the A site on the bacterial ribosome, and chloramphenicol blocks peptidyl transfer. What specific effect would you expect each of these antibiotics to have on protein synthesis?

Tetracycline would directly affect:

1.  tRNA binding to the ribosome
2.  ribosome assembly
3.  growth of the protein chain
{: data-number-style="lower-alpha"}

Chloramphenicol would directly affect

1.  tRNA binding to the ribosome
2.  ribosome assembly
3.  growth of the protein chain
{: data-number-style="lower-alpha"}

</div>
<div data-type="solution" markdown="1">
[[link]](#fig-ch15_05_02) Tetracycline: a; Chloramphenicol: c.

</div>
</div>

# Review Questions

<div data-type="exercise">
<div data-type="problem" markdown="1">
The RNA components of ribosomes are synthesized in the ________.

1.  cytoplasm
2.  nucleus
3.  nucleolus
4.  endoplasmic reticulum
{: data-number-style="lower-alpha"}

</div>
<div data-type="solution" markdown="1">
C

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
In any given species, there are at least how many types of aminoacyl tRNA synthetases?

1.  20
2.  40
3.  100
4.  200
{: data-number-style="lower-alpha"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

# Free Response

<div data-type="exercise">
<div data-type="problem" markdown="1">
Transcribe and translate the following DNA sequence (nontemplate strand): 5'-ATGGCCGGTTATTAAGCA-3'

</div>
<div data-type="solution" markdown="1">
The mRNA would be: 5'-AUGGCCGGUUAUUAAGCA-3'. The protein would be: MAGY. Even though there are six codons, the fifth codon corresponds to a stop, so the sixth codon would not be translated.

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Explain how single nucleotide changes can have vastly different effects on protein function.

</div>
<div data-type="solution" markdown="1">
Nucleotide changes in the third position of codons may not change the amino acid and would have no effect on the protein. Other nucleotide changes that change important amino acids or create or delete start or stop codons would have severe effects on the amino acid sequence of the protein.

</div>
</div>



[1]: http://openstaxcollege.org/l/prokary_protein
